test_container/tests/test/java/timezone.py

```diff
@@ -0,0 +1,66 @@
+# !/usr/bin/env python3
+
+from exasol_python_test_framework import udf
+from exasol_python_test_framework.udf import requires
+
+
+class TimeZoneTest(udf.TestCase):
+
+    def setUp(self):
+        self.query('DROP SCHEMA tz_java CASCADE', ignore_errors=True)
+        self.query('CREATE SCHEMA tz_java')
+
+    def test_default(self):
+        rows = self.query('''
+                          SELECT DBTIMEZONE
+                          FROM DUAL
+                          ''')
+        self.assertRowsEqual([("EUROPE/BERLIN",)], rows)
+        self.query(udf.fixindent('''
+        CREATE OR REPLACE java SCALAR SCRIPT
+        default_tz()
+        RETURNS VARCHAR(100) AS
+        import java.time.ZoneId;
+        import java.util.TimeZone;
+
+        class DEFAULT_TZ{
+            static String run(ExaMetadata exa, ExaIterator ctx) throws Exception {
+                TimeZone timeZone = TimeZone.getDefault();
+                return timeZone.getDisplayName(false, TimeZone.SHORT);
+
+            }
+        }
+        /
+        '''))
+        rows = self.query('''
+                          SELECT tz_java.default_tz()
+                          FROM DUAL
+                          ''')
+        self.assertRowsEqual([("CET",)], rows)
+
+    def test_set_tz(self):
```

Comment:
I think, having additional the test with the jvm parameter is a good idea

Reply:
Done